decidim-core/app/presenters/decidim/log/user_presenter.rb

```diff
@@ -46,6 +46,7 @@ def present
       # Returns an HTML-safe String.
       def present_user
         return h.content_tag(:span, present_user_name, class: "logs__log__author") if user.blank?
+        return "Deleted" if user.respond_to?(:deleted?) && user.deleted?
```

Comment:
I think you should add an i18n string here.
```suggestion
        return I18n.t("decidim.profile.deleted") if user.respond_to?(:deleted?) && user.deleted?
```